modules/rhel-compute-overview.adoc

```diff
@@ -8,18 +8,18 @@
 [id="rhel-compute-overview_{context}"]
 = About adding RHEL compute nodes to a cluster
 
-In {product-title} {product-version}, you have the option of using Red Hat Enterprise Linux (RHEL) machines as compute machines, which are also known as worker machines, in your cluster if you use a user-provisioned infrastructure installation. You must use {op-system-first} machines for the control plane, or master, machines in your cluster.
+In {product-title} {product-version}, you have the option of using {op-system-base-full} machines as compute machines in your cluster if you use a user-provisioned infrastructure installation on the `x86_64` architecture. You must use {op-system-first} machines for the control plane machines in your cluster.
 
-As with all installations that use user-provisioned infrastructure, if you choose to use RHEL compute machines in your cluster, you take responsibility for all operating system life cycle management and maintenance, including performing system updates, applying patches, and completing all other required tasks.
+As with all installations that use user-provisioned infrastructure, if you choose to use {op-system-base} compute machines in your cluster, you take responsibility for all operating system life cycle management and maintenance, including performing system updates, applying patches, and completing all other required tasks.
```

Comment:
This sentence is longer than the IBM Style Guide's 32 word recommendation. What do you think of breaking it up a little? Also, do to the complexity of the content, I find it easier to track without the gerunds.

This is just an idea, feel free to do with it what you will: 
```suggestion
If you choose to use {op-system-base} compute machines in your cluster, you are responsible for all operating system life cycle management and maintenance. You must perform system updates, apply patches, and complete all other required tasks with/ user-provisioned infrastructure installations.
```

Reply:
```suggestion
With installations that use user-provisioned infrastructure, you take responsibility for all operating system life cycle management and maintenance. If you choose to use {op-system-base} compute machines in your cluster, you must perform system updates, apply patches, and complete all other required tasks.
```
Another idea for rewording :)

Reply:
Agree it's easier to read. I even left out UPI now. With the mention of UPI only in the previous sentence I think that's sufficient. @michaelryanpeter WDYT?